Dockerfile

```diff
@@ -19,6 +19,10 @@ ADD . /src/github.com/openwhisk/openwhisk-cli
 RUN echo "Restoring Go dependencies"
 RUN cd /src/github.com/openwhisk/openwhisk-cli && /bin/godep restore -v
 
+# Collect all translated strings into single .go module
+RUN echo "Packaging i18n Go module"
+RUN cd /src/github.com/openwhisk/openwhisk-cli && /bin/go-bindata -pkg wski18n -o wski18n/i18n_resources.go wski18n/resources
```

Comment:
It used to be necessary to have them, because GO CLI within openwhisk is not in a standard go project structure. After we put in a standard go project structure, we do not need to build the module, which can be accessed by the source code instead. All the package imports have been changed to use the absolute paths.